main.go

```diff
@@ -279,52 +279,66 @@ func getPolicies() {
 	}
 }
 
+// stripSlashes removes any trailing slashes from the request's URL
+// path.
```

Comment:
nit

Reply:
?

Reply:
necessary to split the line for an extra couple of characters over length to be displayed in gh window? https://github.com/golang/go/wiki/CodeReviewComments#line-length

Reply:
It's not about github - I just wrap to 72 chars as a general rule. Perhaps not necessary here, but I'm consistent and it's automated.

Reply:
If we are doing that, then would be good to build into some kind of linter as pattern is not followed everywhere by everyone so no consistency?

Reply:
I meant consistency with myself :) Perhaps a linter would be a good idea, if it was restricted to godocs. I don't think there is a tool for that.